packages/snaps-rpc-methods/src/restricted/getBip32Entropy.test.ts

```diff
@@ -200,6 +220,52 @@ describe('getBip32EntropyImplementation', () => {
       const getMnemonic = jest
         .fn()
         .mockImplementation(() => TEST_SECRET_RECOVERY_PHRASE_BYTES);
+      const getMnemonicSeed = jest
+        .fn()
+        .mockResolvedValue(TEST_SECRET_RECOVERY_PHRASE_SEED_BYTES);
+
+      const getUnlockPromise = jest.fn();
+      const getClientCryptography = jest.fn().mockReturnValue({});
+
+      expect(
+        await getBip32EntropyImplementation({
+          getUnlockPromise,
+          getMnemonic,
+          getMnemonicSeed,
+          getClientCryptography,
+        })({
+          method: 'snap_getBip32Entropy',
+          context: { origin: MOCK_SNAP_ID },
+          params: {
+            path: ['m', "44'", "1'"],
+            curve: 'ed25519Bip32',
+            source: 'source-id',
+          },
+        }),
+      ).toMatchInlineSnapshot(`
+        {
+          "chainCode": "0x25ebfafe30b0178f7b95b9a8580c386800344f4820697cc5e8cac4afb6d91d01",
+          "curve": "ed25519Bip32",
+          "depth": 2,
+          "index": 2147483649,
+          "masterFingerprint": 1587894111,
+          "network": "mainnet",
+          "parentFingerprint": 1429088440,
+          "privateKey": "0x98a1e11f532dc7c8130246d6bce0bea5412cb333862f7de2cf94cbaedfbc734e7c6bcc37dd7270a54709ff022dbae6b9fa416a25ac760ccfeff157a469eab0a3",
+          "publicKey": "0x5af28295d600e796ed0b4e51ec8a0f3d5b1f8a647d3ba7a56d7eb0941561d537",
+        }
+      `);
+
+      expect(getMnemonic).toHaveBeenCalledWith('source-id');
```

Comment:
```suggestion
      expect(getMnemonic).toHaveBeenCalledWith('source-id');
      expect(getMnemonicSeed).not.toHaveBeenCalled();
```

In other test files too.

Reply:
Added!